apps/marketplace/main.scss

```diff
@@ -10,6 +10,14 @@
   margin-bottom: 3rem;
 }
 
+.margin-bottom-4x {
```

Comment:
This style and some others aren't used.  Do we need them?

Reply:
added so we don't have gaps